docs/spec/namedtuples.rst

```diff
@@ -0,0 +1,147 @@
+Named Tuples
+============
+
+As with tuples, named tuple classes require some special behaviors for type
+checking.
+
+
+Defining Named Tuples
+---------------------
+
+A named tuple class can be defined using a class syntax or a function syntax.
```

Comment:
Thank you for writing this up. I have a tiny nit about the terminology "function syntax" used throughout this page. For me this term is associated with the `def` statement, I feel it describes the syntax used to write functions in python. Here however named tuples are created with a function call. Would "function-call syntax" or maybe "functional syntax" be a better term here?

Reply:
That's a good point. The typing docs use the term "functional syntax" and the `namedtuple` documentation calls it a "factory function call". I think either of these work, but I've opted for the latter.